src/tables.ts

```diff
@@ -9,6 +9,138 @@ import { RuntimeError } from './util';
 
 const RESOLUTION = 16384;
 
+export const CommonMetrics = {
+  fontFamily: 'Bravura',
+  fontSize: 30,
+  fontWeight: 'normal',
+  fontStyle: 'normal',
+
```

Comment:
This block at the top. Does it serve as default values for fontFamily, fontSize, etc?

I find it interesting that we are now combining text font and music font handling into one. That's cool... but it does seem odd that FretHandFinger's `Arial, sans-serif` will override the default `Bravura`.... since they very different. Arial is for rendering text, while Bravura is for rendering music glyphs.

Reply:
Yes, you will see that the combination will spread over the future classes. I have overriden the defaults to make sure that there are no visual differences. After this PR, the next, chordsymbols, will come with visual differences.